### PR TITLE
issue_58: Job Requests

### PR DESCRIPTION
**Pull request by NuwanJ**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -15,14 +15,18 @@
...
  * These routes can only be accessed by users with type `admin`
  */
-Route::group(['prefix' => 'admin', 'as' => 'admin.', 'middleware' => 'admin'], function () {
-    includeRouteFiles(__DIR__.'/backend/');
-});
+//Route::group(['prefix' => 'admin', 'as' => 'admin.', 'middleware' => 'admin'], function () {
+//    includeRouteFiles(__DIR__ . '/backend/');
+//});
+
+Route::prefix('admin')->as('admin.')->middleware(['auth'])->group(function () {
```
Comment (ishanfdo18098) [inline routes/web.php:30]: Middleware should be 'admin' ?

File: resources/views/frontend/includes/nav.blade.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ class="nav-link dropdown-toggle"
                         </x-utils.link>
 
                         <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-                            @if ($logged_in_user->isAdmin())
+                            @if ($logged_in_user->isAdmin() || $logged_in_user->isLecturer() || $logged_in_user->isTechnicalOfficer() || $logged_in_user->isMaintainer() )
```
Comment (ishanfdo18098) [inline resources/views/frontend/includes/nav.blade.php:68]: isTechnicalOfficer or isTechOfficer ?

File: resources/views/backend/auth/role/create.blade.php [hunk 3]
```diff
@@ -24,6 +24,9 @@
                             <select name="type" class="form-control" required x-on:change="userType = $event.target.value">
                                 <option value="{{ $model::TYPE_USER }}">@lang('User')</option>
                                 <option value="{{ $model::TYPE_ADMIN }}">@lang('Administrator')</option>
+                                <option value="{{ $model::TYPE_LECTURER }}">@lang('Lecturer')</option>
+                                <option value="{{ $model::TYPE_TECH_OFFICER }}">@lang('TechOfficer')</option>
+                                <option value="{{ $model::TYPE_Maintainer }}">@lang('Maintainer')</option>
```
Comment (ishanfdo18098) [inline resources/views/backend/auth/role/create.blade.php:29]: TECH_MAINTAINER in uppercase ?

**Comment by NuwanJ** [comment]
I didn't go through every line for the code review. Hope this is fine. 
Have a lot to implement, but will use a separate ticket for it.

**Pull request merged**